__tests__/app.test.js

```diff
@@ -32,4 +32,26 @@ describe("App", () => {
         });
     });
   });
+  describe("GET /api/reviews", () => {
+    test(`200: Responds with and array of review objects, each of which should have an owner, title, review_id, category, review_img_url, created_at, votes, designer, and comment_count property`, () => {
+      return request(app)
+        .get("/api/reviews")
+        .expect(200)
+        .then(({ body, body: { reviews } }) => {
```

Comment:
Unless i'm missing something here, I think this would just work as { body: { reviews }}

Reply:
You cannot, it spits out a ref error. I think it's because it's renaming(?) the variable with an object? Like for what I did with { rows: reviews }